cloudwatchevents/guardduty/packaged.yaml

```diff
@@ -7,32 +8,31 @@ Description: 'This function is invoked by AWS CloudWatch events in response to s
 Globals:
   Function:
     Timeout: 300
-Outputs:
-  CloudWatchEventFunction:
-    Description: CloudWatchEvent Processor Function ARN
-    Value:
-      Fn::GetAtt:
-      - CloudWatchEventFunction
-      - Arn
 Parameters:
   SumoEndpointUrl:
     Type: String
 Resources:
   CloudWatchEventFunction:
+    Type: AWS::Serverless::Function
     Properties:
-      CodeUri: s3://appdevstore/e62e525a25bb080e521d8bf64909ea41
+      CodeUri: s3://appdevstore/GuardDuty/1.0.2/0d13904d1fbd69ecd879992058f1aa41
```

Comment:
naming is not consistent, see LogGroupConnector has "v" in the version ("v1.0.1")